pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/UploadStatsLoader.kt

```diff
@@ -393,6 +382,33 @@ class UploadStatsLoader: KoinComponent {
         """.trimIndent()
     }
 
+
+    /**
+     * Builds a SQL query to retrieve the uploads with an item for blob-file-copy and a status of failure.
+     *
+     * @param dataStreamId The ID of the data stream.
+     * @param dataStreamRoute The route of the data stream.
+     * @param timeRangeWhereClause The SQL clause for the time range.
+     * @return The SQL query string.
+     */
+    private fun buildBlobFileCopyFailureQuery1(
```

Comment:
Can buildBlobFileCopyFailureQuery1 be removed?  I don't see that it is used anywhere.

Reply:
Yes. It was a copy/paste error. Removed.